### ISSUE TITLE
PubMed Data Set Query Option Missing in PyPI Version of Alexandria3k

### ISSUE THREAD

**Issue by panos-span** [dspinellis/alexandria3k#42]
**Environment**:
 - OS : Ubuntu 22.04 (Windows Subsystem for Linux 2) 
 - Python Version : 3.11.5

**Alexandria3k Version**: 3.1.2

**Issue Description**:

When attempting to perform a query on the pubmed-data using the alexandria3k CLI, the system returns an error indicating that 'pubmed-data' is not a valid choice for data_name and it doesn't seem like there is a choice for 'pubmed-data' in general. This occurs despite the package documentation suggesting that PubMed data set queries are supported.

**Command Attempted**:
~~~bash
panaspanakis@hpc:~$ a3k query pubmed-data 'pubmed_data' --query 'SELECT DOI, title FROM pubmed_articles WHERE published_year > 2020 AND title LIKE "%Machine Learning%"'
~~~

**Error Received**:
~~~bash
usage: a3k query [-h] [-a ATTACH_DATABASES [ATTACH_DATABASES ...]] [-E OUTPUT_ENCODING] [-F FIELD_SEPARATOR] [-H] [-o OUTPUT] [-P] (-Q QUERY_FILE | -q QUERY) [-s SAMPLE]
                 {orcid,doaj,asjcs,journal-names,uspto,crossref,ror,funder-names} [data_location]
a3k query: error: argument data_name: invalid choice: 'pubmed-data' (choose from 'orcid', 'doaj', 'asjcs', 'journal-names', 'uspto', 'crossref', 'ror', 'funder-names')
~~~

Here is also a screenshot of the described issue below:

![image](https://github.com/dspinellis/alexandria3k/assets/93033399/eeb6c634-9942-46b2-a8bf-2e1aa6f810c0)

**Comment by dspinellis** [comment]
😃

**Comment by dspinellis** [comment]
Fixed by releasing version 3.2.0.
